### PR TITLE
Update s3 docs

### PR DESCRIPTION
**Pull request by calvinjia**
Improves the flow of the s3 documentation.

### CODE REVIEW

File: docs/en/ufs/S3.md [hunk 1]
```diff
@@ -23,26 +23,22 @@ should also note the directory you want to use in that bucket, either by creatin
...
+## Basic Setup
 
 Alluxio unifies access to different storage systems through the
 [unified namespace]({{ '/en/advanced/Namespace-Management.html' | relativize_url }}) feature.
-An S3 location can be either mounted at
-the root of the Alluxio namespace or at a nested directory.
+An S3 location can be either mounted at the root of the Alluxio namespace or at a nested directory.
 
-### Root Mount
+### Root Mount Point
```
Comment (apc999) [inline docs/en/ufs/S3.md]: for the part of describing different ways to set credential after "Specify the AWS credentials for S3 access.", can we move them to the `Advanced Setup` session, but using setting `aws.accessKeyId` and `aws.secretKey` in site properties in this `Basic Setup` session?

Comment (calvinjia) [reply docs/en/ufs/S3.md]: Updated, thanks.

File: docs/en/ufs/S3.md [hunk 2]
```diff
@@ -23,33 +23,85 @@ should also note the directory you want to use in that bucket, either by creatin
...
+Configure Alluxio to use S3 as its under storage system by modifying `conf/alluxio-site.properties`.
+Specify an **existing** S3 bucket and directory as the under storage system by modifying
 `conf/alluxio-site.properties` to include:
 
 ```
 alluxio.underfs.address=s3a://S3_BUCKET/S3_DIRECTORY
 ```
 
-Specify the AWS credentials for S3 access.
+Specify the AWS credentials for S3 access by setting aws.accessKeyId and aws.secretKey in
```
Comment (apc999) [inline docs/en/ufs/S3.md]: nit: quote aws.accessKeyId and aws.secretKey

File: docs/en/ufs/S3.md [hunk 3]
```diff
@@ -23,33 +23,85 @@ should also note the directory you want to use in that bucket, either by creatin
...
+
+```bash
+$ ./bin/alluxio-stop.sh local
+```
+
+## Advanced Setup
+
+### Advanced Credentials Setup
 
 You can specify credentials in different ways, from highest to lowest priority:
```
Comment (apc999) [inline docs/en/ufs/S3.md]: quote the following key words like aws.accessKeyId and aws.secretKey ?

**Comment by apc999** [comment]
Minor comments left. Otherwise, LGTM

**Pull request merged**